### ISSUE TITLE
Blockstun Reversal Issues Win 11

### ISSUE THREAD

**Issue by Superjig** [Iquis/rev2-wakeup-tool#37]
OS Details:
```
Edition	Windows 11 Pro
Version	21H2
Installed on	‎04/‎03/‎2022
OS build	22000.675
Experience	Windows Feature Experience Pack 1000.22000.675.0
```

Blockstun Reversal is not working at all for ASM injection (neither is Wakeup Reversal) but indicate a succes in logs:
```
[2022-05-29 16:06:00] Trigger Replay by Asm Injection
[2022-05-29 16:06:00] Reversal Done!
[2022-05-29 16:06:03] Reversal!

```
It only works around half the time for keyboard stroke method, and the inputs that don't come out aren't logged.
Presumably an issue with polling rate? Seems about 50/50 of them dropping.

Restarted multiple times, windows without any unnecessary background applications.

**Comment by DevinSova** [comment]
Same, having the same issue on Windows 11 Pro

**Comment by ghost** [comment]
Been a while so unsure if work was done that might have caused regressions, but I cannot get the tool to work at all on Windows 11 now. Tried running in compatibility mode and tried every included option in the tool and despite the old pre-fork wakeup dp tool working, this one doesn't seem to.

**Edit 9/17/23: Tried reinstalling on someone's advice once again and this time it's working for reasons I can't explain.

**Comment by worsety** [comment]
Should be fixed as 1.2 onward use a different reversal method, please retest.
